KubeArmor/core/kubeUpdate.go

```diff
@@ -2607,8 +2684,14 @@ func (dm *KubeArmorDaemon) WatchDefaultPosture() {
 					CapabilitiesAction: cp,
 				}
 				annotated := fa || na || ca
+				visibility := tp.Visibility{
+					File:         dm.validateVisibility("file", ns, cfg.GlobalCfg.Visibility),
+					Process:      dm.validateVisibility("process", ns, cfg.GlobalCfg.Visibility),
+					Network:      dm.validateVisibility("network", ns, cfg.GlobalCfg.Visibility),
+					Capabilities: dm.validateVisibility("capabilities", ns, cfg.GlobalCfg.Visibility),
```

Comment:
can we directly define the visibility flag in the validatevisibility() func directly, so that we are not calling it every time?

Reply:
I am not sure what do you mean by this, Can you explain?

Reply:
File: dm.validateVisibility("file", ns) and keep cfg.GlobalCfg.Visibility inside the function itself.